### PR TITLE
feat(TDI-47598): Kafka support version 3.2

### PR DESCRIPTION
**Pull request by olhava**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by ozhelezniak-talend — DISMISSED** [review]
LGTM

File: main/plugins/org.talend.designer.components.bigdata/components/tKafkaCreateTopic/2.4.x/tKafkaCreateTopic_begin_2.4.x.javajet [hunk 1]
```diff
@@ -24,6 +24,15 @@ java.util.Map<String, String> <%=cid%>_topicProperties = new java.util.HashMap<>
 
 java.util.Map<String, Object> <%=cid%>_clientConf = new java.util.HashMap<>();
 <%=cid%>_clientConf.put(org.apache.kafka.clients.consumer.ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG,<%=tKafkaCreateTopicUtil.getKafkaConnect()%>);
+
+<%
+    for(Entry<String, String> kafkaProperty : tKafkaCreateTopicUtil.getKafkaProperties().entrySet()) {
```
[the remark this answers is not in the document]
Comment (olhava) [reply main/plugins/org.talend.designer.components.bigdata/components/tKafkaCreateTopic/2.4.x/tKafkaCreateTopic_begin_2.4.x.javajet:29]: Rechecked this case. No compilation errors in generated code.
There is a separate map with a name that starts with a component id:
```
tKafkaCreateTopic_1_clientConf.put("key1", "value1");
...
tKafkaCreateTopic_1_clientConf.put("key1", "value1");
```

File: main/plugins/org.talend.designer.components.bigdata/components/tKafkaCreateTopic/tKafkaCreateTopic_util.javajet [hunk 2]
```diff
@@ -87,5 +95,17 @@ class TKafkaCreateTopicUtil {
 		}
 		return result;
    }
+
+    private Map<String, String> findKafkaProperties() {
```
[the remark this answers is not in the document]
Comment (olhava) [reply main/plugins/org.talend.designer.components.bigdata/components/tKafkaCreateTopic/tKafkaCreateTopic_util.javajet:99]: Agree that the solution with passing the value as a parameter is more transparent.
Current solution was chosen for a better consistency with other functionality in this and other related components.

**Review by ozhelezniak-talend — DISMISSED** [review]
lgtm

**Review by ozhelezniak-talend — APPROVED** [review]
lgtm (no file changed after rebase)

**Pull request merged**